src/com/company/Main.java

```diff
@@ -74,17 +76,56 @@ private static void printMenu() {
 
     }
 
-    private static void zeroIfNegative()
-    {
+    private static void zeroIfNegative() {
         System.out.println("Zero If Less Than Negative");
-        if ( isString ){
+        if (isString) {
             System.out.print(inputString);
             System.out.println("\n=======================================");
             return;
         }
-        for ( int i=0;i<size;i++){
-            System.out.print(arr[i]<0?0:arr[i]+" ");
+        for (int i = 0; i < size; i++) {
+            System.out.print(arr[i] < 0 ? 0 : arr[i] + " ");
         }
         System.out.println("\n=======================================");
     }
+
+    /**
+     * Counts the number of primes in the global array arr.
+     */
+    private static void countPrime() {
+        System.out.println("Primes Count");
+
+        boolean prime = true;
+        int count = 0;
+
+        int[] numbers = arr;
```

Comment:
Here you are doing a shallow copy which is ok for now but when execute all is executed the input array may change during the execution of any function
try execute all now to see what I mean